lib/vquic/curl_ngtcp2.c

```diff
@@ -2742,8 +2742,9 @@ static bool cf_ngtcp2_conn_is_alive(struct Curl_cfilter *cf,
    * it will close the connection when it expires. */
   rp = ngtcp2_conn_get_remote_transport_params(ctx->qconn);
   if(rp && rp->max_idle_timeout) {
-    timediff_t idletime = curlx_timediff(curlx_now(), ctx->q.last_io);
-    if(idletime > 0 && (uint64_t)idletime > rp->max_idle_timeout)
+    timediff_t idletime_ms = curlx_timediff(curlx_now(), ctx->q.last_io);
+    if(idletime_ms > 0 &&
+      ((uint64_t)idletime_ms * NGTCP2_MILLISECONDS) > rp->max_idle_timeout)
```

Comment:
To avoid triggering integer overflow thoughts here, wouldn't it be easier to instead devide the the `max_idle_timeout` with NGTCP2_MILLISECONDS instead of multiplying the idletime?

Reply:
Yup. Will fix this + the formatting typo